test/contracts/HCElection.aes

```diff
@@ -3,6 +3,7 @@ include "Pair.aes"
 
 contract interface MainStaking =
   entrypoint sorted_validators : () => list((address * int))
+  entrypoint lock_stake  : (int) => list((address * int))
```

Comment:
Not important, but would `locked_stake` not be a better name for the stake locked at that epoch?
Whereas `lock_state` suggests that you do lock state for that epoch?

Reply:
We do lock stake for that epoch, but for convenience we also return the stake that was just locked?!